components/Cards/src/MDCCard.m

```diff
@@ -282,4 +302,35 @@ - (void)updateBackgroundColor {
   self.layer.shapedBackgroundColor = _backgroundColor;
 }
 
+- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
+  if ([self.rippleDelegate respondsToSelector:@selector(cardRippleDelegateTouchesBegan:withEvent:)]) {
+    [self.rippleDelegate cardRippleDelegateTouchesBegan:touches withEvent:event];
+  }
+  [super touchesBegan:touches withEvent:event];
+}
+
+- (void)touchesMoved:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
+  // The ripple invocation must come before touchesMoved of the super, otherwise the setHighlighted
+  // of the UIControl will be triggered before the ripple identifies that the highlighted was
+  // trigerred from a long press entering the view and shouldn't invoke a ripple.
+  if ([self.rippleDelegate respondsToSelector:@selector(cardRippleDelegateTouchesMoved:withEvent:)]) {
+    [self.rippleDelegate cardRippleDelegateTouchesMoved:touches withEvent:event];
+  }
```

Comment:
I believe you can drop the responds to selector calls here and throughout because the protocol’s methods are all required.

Reply:
Done.